datalake/src/main/java/com/sequenceiq/datalake/service/sdx/dr/SdxBackupRestoreService.java

```diff
@@ -117,6 +126,18 @@ public class SdxBackupRestoreService {
     @Inject
     private EnvironmentClientService environmentClientService;
 
+    private static boolean isVersionOlderThan(SdxCluster cluster, String baseVersion) {
```

Comment:
Why are the privates moved to the front of the class?

Reply:
Because these are static methods.
But I'm going to move these to the end of the class.